client/main/backstage.js

```diff
@@ -108,7 +106,7 @@ const FSEntryListView = SilkyView.extend({
                 location = item.description;
             }
 
-            html += '<div class="silky-bs-fslist-entry" data-path="' + s6e(filePath) + '">';
+            html = `<div class="silky-bs-fslist-entry bs-menu-list-item" data-path="${s6e(filePath)}" tabindex="-1">`;
```

Comment:
can't quite see why, inside a loop, you'd go from `html += ` to `html = ` ... might be correct ...